src/VirtoCommerce.Platform.Modules/ModuleInitializer.cs

```diff
@@ -18,16 +19,22 @@ public class ModuleInitializer : IModuleInitializer
         private readonly ILogger<ModuleInitializer> _loggerFacade;
         private readonly IServiceCollection _serviceCollection;
         private readonly IConfiguration _configuration;
+        private readonly IHostEnvironment _hostingEnvironment;
 
         /// <summary>
         /// Initializes a new instance of <see cref="ModuleInitializer"/>.
         /// </summary>
         /// <param name="loggerFacade">The logger to use.</param>
-        public ModuleInitializer(ILogger<ModuleInitializer> loggerFacade, IServiceCollection serviceCollection, IConfiguration configuration)
+        public ModuleInitializer(
+            ILogger<ModuleInitializer> loggerFacade
+            , IServiceCollection serviceCollection
+            , IConfiguration configuration
+            , IHostEnvironment hostingEnvironment)
```

Comment:
We write it this way everywhere.
```suggestion
        public ModuleInitializer(
            ILogger<ModuleInitializer> loggerFacade,
            IServiceCollection serviceCollection,
            IConfiguration configuration,
            IHostEnvironment hostingEnvironment)
```